src/main/java/Storage.java

```diff
@@ -0,0 +1,68 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Storage {
+    public Storage(){}
```

Comment:
I'm not sure if this constructor serves a purpose as you'd be able to create a Storage object just fine without this. Unless you're setting it up for future modifications.